src/download-package.js

```diff
@@ -18,20 +26,22 @@ async function downloadPackage(name, url, range) {
 
   let newTmpDir = await createTmpDir();
   let { stdout } = await spawn('npm', ['install', url], { cwd: newTmpDir });
-  // if (!name) {
-  //   name = stdout.match(/^\+ (.*)@\d\.\d\.\d.*$/m)[1];
-  // }
   let arg = stdout.match(/^\+ (.*)$/m)[1];
   let parsed = npa(arg);
   name = parsed.name;
   let version = parsed.rawSpec;
   let _path = path.join(newTmpDir, 'node_modules', name);
-  // let version = require(path.join(_path, 'package')).version;
+  let defaultBlueprintOverride;
+
+  if (parsed['ember-addon'] && parsed['ember-addon']['defaultBlueprint']) {
```

Comment:
is ember-addon.defaultBlueprint documented?

Reply:
This must be recent: https://cli.emberjs.com/release/writing-addons/addon-blueprints/#blueprintconfig . I've only learned about it from re-reading the docs about blueprints recently lol